File: API/Helpers/ColUserParams.cs
namespace API.Helpers
{
    public class ColUserParams : ColPaginationParams
    {
        public string CurrentColUsername { get; set; }
        public string ColUserType { get; set; }
        public int MinEnrollment { get; set; } = 300;
        public int MaxEnrollment { get; set; } = 6000;
        public string CollegeLocation { get; set; }
        public string ClassYear { get; set; } = "Junior";
        public string ProposedMajor { get; set; } = "Accounting";
        public string CollegeName { get; set; }
        public string FirstName { get; set; }
        public string OrderBy { get; set; } = "LastActive";
    }
}

File: client/src/app/_models/colUserParams.ts
import { ColUser } from './colUser';

export class ColUserParams {
  colUserType: string;
  minEnrollment = 300;
  maxEnrollment = 6000;
  collegeLocation = 'Des Moines, IA';
  collegeName: string;
  pageNumber = 1;
  pageSize = 3;
  classYear = 'Junior';
  proposedMajor = 'Accounting';

  orderBy = 'lastActive';

  constructor(colUser: ColUser) {
    this.colUserType =
      colUser.colUserType === 'College' ? 'ColLead' : 'College';
  }
}
